impl/src/test/groovy/com/okta/sdk/impl/resource/SessionsTest.groovy

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2017 Okta
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.resource
```

Comment:
Why is `SessionsTest` inside `resource` package?
Is it considered a resource?

Reply:
yeah, it does look confusing there, i'll move it